### DISCUSSION TITLE
Cockpit script

### DISCUSSION THREAD

**Discussion by aelsharawi** [tteck/Proxmox#2683] [Help Needed] [answered]
https://blog.kye.dev/proxmox-zfs

your very helpful guide made me very eager to shift from truenas vm to proxmox

I would appreciate if you can prepare. LXC script that at least put the first easy steps for that shift

one more  will I be able to migrate the pool from truenas without losing its data?
thanks

**Comment by aelsharawi** [comment]
I was able to move from truenas pool (I have to export and destroy the pool) as they copying was not stable enough


-followed your guide and have few questions please


-why I can not add a mount point in the GUI - I was able to do it with the command though

-what about the mount point encryption is it safe to use it

**Reply by tteck** [reply, marked as answer]
>followed your guide and have few questions please

**That is not my guide!**
